app/src/main/java/de/westnordost/streetcomplete/view/ItemViewHolder.kt

```diff
@@ -41,7 +42,7 @@ class ItemViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
             textView?.text = null
 
         val descriptionId = item.descriptionId
-        descriptionView?.visibility = if (descriptionId != null) View.VISIBLE else View.GONE
+        descriptionView?.isVisible = descriptionId != null
```

Comment:
I am not sure if I am a fan of that extension function because it hides away the information between which states (visible / gone vs visible / invisible) it switches.

I had to look in the [documentation](https://developer.android.com/reference/kotlin/androidx/core/view/package-summary#isvisible) to see what it does exactly.  I think exclusively using `isInvisible` and `isGone` would be better because it is explicit between which states it alternates

Reply:
> I had to look in the [documentation](https://developer.android.com/reference/kotlin/androidx/core/view/package-summary#isvisible) to see what it does exactly. I think exclusively using `isInvisible` and `isGone` would be better because it is explicit between which states it alternates

Okay, I'll remove usages of `isVisible`.